react/features/base/lib-jitsi-meet/native/Storage.js

```diff
@@ -88,6 +88,28 @@ export default class Storage {
         return this.hasOwnProperty(key) ? this[key] : null;
     }
 
+    /**
+     * Returns the value associated with a specific key in this storage in an
+     * async manner. This method is required for those cases where we need
+     * the stored data but we're not sure yet whether the {@code Storage}
+     * is already initialised or not - e.g. on app start.
+     *
+     * @private
+     * @param {string} key - The name of the key to retrieve the value of.
+     * @param {Function} cb - The callback to be invoked when the
+     * data is available.
+     * @returns {Promise}
+     */
+    _getItemAsync(key) {
+        return new Promise(resolve => {
+            AsyncStorage.getItem(
+                `${String(this._keyPrefix)}${key}`,
+                (error, result) => {
+                    resolve(result ? result : null);
```

Comment:
IIRC browsers return `undefined` for non-existing keys, I think we should do the same here.

Reply:
The other implementation if getItem returns null if the key was not found so I thought it should do the same. Should I change it?

Reply:
https://developer.mozilla.org/en-US/docs/Web/API/Storage/getItem says: If the key does not exist, null is returned.